### ISSUE TITLE
Automated curation of twitter feed as posts for community pages

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#176]
Twitter posts are treated as a custom content type, but they are manually generated. Look to see if we can provide a more streamlined way to ingest and curate these, perhaps this plugin - http://wordpress.org/plugins/twitter-posts-to-blog/

See also #75

**Comment by JeanneHolm** [comment]
Merge with #108

**Comment by philipashlock** [comment]
I think this since it's distinct from #108. You mentioned that 108 is about providing share buttons for users on the site. This ticket is for integrating a data.gov related twitter account in wordpress for posting tweets as posts.

**Comment by ErieMeyer** [comment]
Totally agree. So, if a USG-confirmed account (https://twitter.com/whitehouse/usg) tweets something with #OpenData, having it dynamically show up on data.gov (with the option to delete if necessary). Would be fantastic.

**Comment by JeanneHolm** [comment]
Echoing the thread on #247 to make Twitter posts easier if possible when they are just individual tweets that from time to time should be shared in addition to automatically including Twitter posts from sources like @HealthDataGov or as @ErieMeyer says USG posts with #opendata

**Comment by Raseman** [comment]
I would suggest we make sure that the twitter handle, twitter avatar, name, tweet, date, and time fields are all visible in the new thing we build.

**Comment by dialsunny** [comment]
The plug-in has been added. 

Note that the plug-in imports tweets as “standard” post-type and not as “status” post-type. Also, the plug-in doesn't allow adding “Topic”  tags to the imported tweet automatically. So, we imported these tweets in Draft form. A publisher can add tweets to his/her Topic and publish. They can filter all tweets for an account and perform bulk operation.

In future, we can explore options to customize the plug-in to associate to a Topic, auto publish, and customize display as needed.
